addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -143,7 +143,7 @@ var FileWidget = SearchableMediaWidget.extend({
             firstFilters: [],
             lastFilters: [],
             showQuickUpload: config.isDebug(),
-            mediaWidth: media && media.clientWidth,
+            mediaWidth: media && media.parentElement && media.parentElement.clientWidth,
```

Comment:
@sdegueldre `clientWidth` returns the width including the paddings, I guess it would be better to use jquery's `width()` to have the correct value. I also feel that using `parentNode` would be more common than `parentElement` but whatever.